README.md

```diff
@@ -103,9 +103,10 @@ Meeting notes are available as a public [Google doc](https://docs.google.com/doc
 
 Approvers ([@open-telemetry/python-approvers](https://github.com/orgs/open-telemetry/teams/python-approvers)):
 
-- [Alex Boten](https://github.com/codeboten), LightStep
 - [Carlos Alberto Cortez](https://github.com/carlosalberto), LightStep
+- [Chris Kleinknecht](https://github.com/c24t), Google
 - [Christian Neumüller](https://github.com/Oberon00), Dynatrace
+- [Diego Hurtado](https://github.com/ocelotl)
```

Comment:
@ocelotl do you want an affiliation here? Let me know if so or feel free to add it yourself.